content/docs/get-started/aws/deploy-stack.md

```diff
@@ -97,6 +97,10 @@ $ pulumi stack output BucketName
 
 {{% /choosable %}}
 
-Running that command will print out the name of your bucket. Now that your bucket has been provisioned, let's modify the bucket to host a static website.
+Running that command will print out the name of your bucket. If you are using a Pulumi managed backend, you can follow the Permalink displayed at the end of the CLI output. This will open the update in the Pulumi [Console](https://app.pulumi.com), where you can view the output and explore detailed information about your stack such as its activity, resources and configuration.
```

Comment:
```suggestion
Running that command will print out the name of your bucket. If you are using the Pulumi Service backend, you can follow the Permalink displayed at the end of the CLI output. This will open the update in the Pulumi [Console](https://app.pulumi.com), where you can view the output and explore detailed information about your stack such as its activity, resources and configuration.
```

Also:
1. Should we put this in a "note" section instead of just in the paragraph?  I feel like that might both make it stand out more and also be more clear that this is sort of "optional".
2. I think the phrasing here is not true relative to the new way the `pulumi` CLI outputs this?  We now say "View Live" instead of "Permalink" and it's at the beginning instead of end.
3. Do we want to link out on "Pulumi Service backend" to the documentation about the backend?  (Not sure we have something "great" to link to - which is another suggestion to improve!)

Reply:
> I think the phrasing here is not true relative to the new way the pulumi CLI outputs this? We now say "View Live" instead of "Permalink" and it's at the beginning instead of end.

yes very true.

> Do we want to link out on "Pulumi Service backend" to the documentation about the backend? (Not sure we have something "great" to link to - which is another suggestion to improve!)

I think that is a good idea. This is the only [page](https://www.pulumi.com/docs/intro/console/) I can think of to link, but I don't think it does a great job of showcasing the console in an organized fashion. I think we may need a console page that is more marketing oriented that has that captures more visuals, i.e. screenshots,  whereas this one seems to be more documentation oriented - unless there is one that exists already, but I am just not aware of it.